cmd/metricsexporter/metricsExporter.go

```diff
@@ -0,0 +1,87 @@
+package main
```

Comment:
is camel case file name `metricsExporter.go`  a standard file name convention in go?  I think it should be `metrics_exporter.go`

Reply:
changed it to `metrics_exporter.go`